### PR TITLE
Update client-access-rules.md

### PR DESCRIPTION
**Pull request by domooney**
Client Access Rules are evaluated after authentication and cannot be used to block raw connection or authentication attempts.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/client-access-rules/client-access-rules.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ How multiple rules with the same condition are evaluated, and how a rule with mu
 
 You can test how a specific client connection would be affected by Client Access Rules (which rules would match and therefore affect the connection). For more information, see [Use Exchange Online PowerShell to test Client Access Rules](procedures-for-client-access-rules.md#use-exchange-online-powershell-to-test-client-access-rules).
 
+> [!NOTE]
+> Client Access Rules are evaluated after authentication and cannot be used to block raw connection or authentication attempts..
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/client-access-rules/client-access-rules.md:76]:
```suggestion
> Client Access Rules are evaluated after authentication and cannot be used to block raw connection or authentication attempts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @domooney, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**